### PR TITLE
added recipe for hboie/jasper-report-bundle

### PR DESCRIPTION
**Pull request by hboie**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by hboie** [comment]
> 
> 
> The diff is empty here.

Dear Nicolas,

this is my first try, to add a recipe - I tried to change the version of my package in the recipe, but obviously I just removed the old version but did not add the new version.

Sorry about this... (I was so happy that the checks were all ok - but this is easy if everything is empty.. :-( )

Best regards,

Hans

**Comment by hboie** [comment]
Dear Nicolas,

I hope that it looks better now...

Best regards,

Hans

File: hboie/jasper-report-bundle/1.1/config/packages/jasper_report.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+hboie_jasper_report:
+    host:      'http://localhost:8080/jasperserver'
+    username:  jasperadmin
+    password:  jasperadmin
```
Comment (Nyholm) [inline hboie/jasper-report-bundle/1.1/config/packages/jasper_report.yaml]: Maybe add environment variables for host, username, password and org_id?

Does that make sense?

**Comment by Nyholm** [comment]
Friendly ping

**Comment by Nyholm** [comment]
Friendly ping

**Comment by Nyholm** [comment]
Im closing this due inactivity. Feel free to reopen this PR or to create a new one.

**Pull request closed without merging**